docs/admin/guides/hosting_site_without_https.md

```diff
@@ -0,0 +1,4 @@
+# Hosting site without https
+
+The default setup is http. In order to host your site without https, 
```

Comment:
The https is provided by the reverse proxy. The default installation of DTaaS runs in http mode.